requirements.txt

```diff
@@ -32,16 +32,9 @@ matplotlib==3.5.3; python_version > '3.6'
 # License: https://github.com/matplotlib/matplotlib/blob/master/LICENSE/LICENSE
 # License: PSF
 
-numpy==1.21.6; python_version <= '3.7'
 numpy==1.23.2; python_version > '3.7'
```

Comment:
This looks to me like numpy would not be installed for Python 3.7 (e.g. from the Ansys EM shipped version).  This could cause a failure except that its required in setup.py without a version number so maybe this one is redundant?

Reply:
@isaacansys That is an excellent point. Yes, it should be fine thanks to the condition in the requirements.
But as I said, I agree with you that it could be confusing. I will try to clarify our installation.
Thanks again for the feedback.

Reply:
@MaxJPRey not sure how standard this would be but for my own smaller projects I have been leaving out requirements.txt entirely so that setup.py or setup.cfg is the only source of build and install requirements.  Then I have a requirements-dev, requirements-docs, etc. for specific additions for development.

Reply:
@isaacansys It was mainly for Dependabot, so it can parse the files properly and inform us of the new releases.